ShareClient/ShareClientManager.swift

```diff
@@ -64,6 +65,14 @@ public class ShareClientManager: CGMManager {
     public var glucoseDisplay: GlucoseDisplayable? {
         return latestBackfill
     }
+    
+    public var cgmStatus: CGMManagerStatus {
+        return CGMManagerStatus(hasValidSensorSession: hasValidSensorSession)
+    }
+
+    public var hasValidSensorSession: Bool {
+        return shareService.isAuthorized
+    }
```

Comment:
Not sure why this indicates the sensor session is valid. Not overly concerned since you know this code well, but think it will cause confusion to any future development. Maybe add a comment to explain why?

Reply:
+1